### ISSUE TITLE
iPhone 7 Restore Syscfg

### ISSUE THREAD

**Issue by IamJoker03** [j4nf4b3l/MagicCFG#3]
iPhone 7 restore syscfg stuck at adding NvSn
![Uploading 60194B33-DFB9-4E46-9B72-C329F39E801A.png…]()

**Comment by IamJoker03** [comment]
![E2649210-132F-40E8-8317-6F3ECF0362F2](https://user-images.githubusercontent.com/34206615/124402237-9a3eec00-dd4c-11eb-9b4f-96d94f0f1fe0.png)

**Comment by bns34** [comment]
I may be wrong as I'm no expert but your assigning rosegold iPhone 6 to an iPhone 7?...

**Comment by IamJoker03** [comment]
Nope, I used a NAND from a 6s iCloud locked board and im trying to restore the iPhone 7's underlying data on it. But not been able to. Happens same with a 128gb iPhone 7 NAND. It gets stuck on the part where it adds NvSn

**Comment by Joebeazelman** [comment]
What cable are you using? Also, do you have the device in DFU mode?

**Comment by ShaviTF** [comment]
Hi... 
    Have you found any solution? I am experiencing exactly the same problem, both with MagicCFG v1.3 and with Purple PRO. In debug mode it does not reflect any information either. Quite simply, when I restore the BackUP, everything seems to go perfectly until it gets to "syscfg add NvSn" and stops. If I disconnect the phone from the cable, it seems to continue until it completes the process apparently, obviously with no phone connected, it has not done anything. I have tried several cables and the same thing happens with all of them.
